guides/source/association_basics.md

```diff
@@ -96,7 +96,7 @@ end
 
 ![belongs_to Association Diagram](images/association_basics/belongs_to.png)
 
-NOTE: `belongs_to` associations _must_ use the singular term. If you used the pluralized form in the above example for the `author` association in the `Book` model, you would be told that there was an "uninitialized constant Book::Authors". This is because Rails automatically infers the class name from the association name. If the association name is wrongly pluralized, then the inferred class will be wrongly pluralized too.
+NOTE: `belongs_to` associations _must_ use the singular term. If you used the pluralized form in the above example for the `author` association in the `Book` model and tried to create the instance by `Book.create(authors: @author)`, you would be told that there was an "uninitialized constant Book::Authors". This is because Rails automatically infers the class name from the association name. If the association name is wrongly pluralized, then the inferred class will be wrongly pluralized too.
```

Comment:
Technically these sentences aren't correct. There is no restriction on naming `belongs_to` association. You can use any name you wish for a `belongs_to` association, for that you only need to pass `class_name` option in order to make it work. This guide explains that http://edgeguides.rubyonrails.org/association_basics.html#options-for-belongs-to-class-name. Maybe we should remove this "NOTE"?

Reply:
@rafaelfranca Thanks.